src/main/resources/templates/snowflake/Makefile.schema.ssp

```diff
@@ -27,13 +27,13 @@ run-all: ${runTables}
 clean-all: ${cleanTables}
 
 #for (table <- tables)
-first-run-${table.destinationName}:
-	$(MAKE) first-run -C ${table.destinationName}
+first-run-${table.getDestinationName}:
+	$(MAKE) first-run -C ${table.getDestinationName}
 
-run-${table.destinationName}:
-	$(MAKE) run -C ${table.destinationName}
+run-${table.getDestinationName}:
```

Comment:
Customers have a bunch of custom templates so they are a public API. This will release in a new major version, so we can change things if we have to. However, I think we want to minimize changes to these templates if at all possible.

Is it possible to make this value public so we don't have to make this change?

Reply:
updated